roles/install_java/tasks/main.yml

```diff
@@ -0,0 +1,23 @@
+---
+- name: Ensure correct Java version is installed - {{ java_package }}
+  ansible.builtin.package:
+    name: "{{ java_package }}-{{ java_package_version }}"
+    state: installed
+
+- name: Set JAVA_HOME through shell script
+  ansible.builtin.template:
+    src: "java_home.sh.j2"
+    dest: "{{ java_profile_d }}/java_home.sh"
+    mode: "0644"
+  when: java_home is defined and java_home != ''
+
+- name: Get info for java package directory
+  ansible.builtin.stat:
+    path: "/usr/lib/jvm/{{ java_package }}"
```

Comment:
`/usr/lib/jvm/{{ java_package }}` is a symlink. We need to use the full path to the target when selecting the version to use with `alternatives`